config.go

```diff
@@ -549,6 +551,63 @@ func (s *Settings) GetSender(ctx context.Context, env Environment) (send.Sender,
 	return send.NewConfiguredMultiSender(senders...), nil
 }
 
+// getGithubAppAuth returns app id and app private key if it exists.
+// returns false if it does not have both of those fields.
+func (s *Settings) getGithubAppAuth() (int64, []byte, bool) {
+	key, ok := s.Expansions[GithubAppPrivateKey]
+	if s.AuthConfig.Github != nil && s.AuthConfig.Github.AppId != 0 && ok && key != "" {
```

Comment:
Can we simplify this if statement down a little bit? Something like:
```go
if s.AuthConfig.Github == nil || s.AuthConfig.Github.AppId == 0 { return no result }
key := s.Expansions[GithubAppPrivateKey]
if key == "" { also return no result }
return successful info
```
It also reads more smoothly when the last return is the success path (i.e. the key is returned), and all the returns before it are the erroneous conditions.